components/dialog/predefined.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Predefined Dialogs
+title: Blazor Dialogs - Alert, Confirm, Prompt | Telerik UI for Blazor
 page_title: Predefined Dialogs - Alert, Confirm, Prompt
```

Comment:
Consider the `page_title` (SEO title) template when implementing requirements by the SEO team.
```suggestion
page_title: Dialogs - Alert, Confirm, Prompt
```